framework/test/mocha/unit/modules/chain/logic/multisignature.js

```diff
@@ -36,6 +40,30 @@ const rawValidTransaction = testData.rawValidTransaction;
 const multiSigAccount1 = testData.multiSigAccount1;
 const multiSigAccount2 = testData.multiSigAccount2;
 
+const createInvalidRegisterMultisignatureTransaction = ({
```

Comment:
Please, move this function to `common/utils/elements`. It's pretty handy to reuse this in future scenarios.

Reply:
This function needs to be removed/changed when we introduce 3.0, and if we're gonna clean up we need to change a lot of stuff not only multiSignature, so i would say leave it as it is for now

Reply:
It's not about cleaning, it's about placing common methods related to the same library into the same file. The same method is being defined twice in different test suites. For reference: https://github.com/LiskHQ/lisk/blob/9e1b61e9a2fc361c0f88771b4f132bc340177543/framework/test/mocha/functional/http/post/4.multisignature.js#L38